src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1436,6 +1296,157 @@ public static function debug_data() {
 		return $info;
 	}
 
+	/**
+	 * Gets the WP constants section of the debug data.
```

Comment:
```suggestion
	 * Gets the WordPress constants section of the debug data.
```
Nit: Let's use the full application name in the docblock.

Reply:
Absolutely. Applied it.